### ISSUE TITLE
Usage instructions

### ISSUE THREAD

**Issue by mirkokg** [bodyshopbidsdotcom/PyWrike#4]
Can you provide some brief usage instructions.

**Comment by santi-h** [comment]
Yeah I'll update the readme this week

**Comment by mirkokg** [comment]
I figured it out 👍 
Looks like method to get list of tasks is not implemented?

**Comment by santi-h** [comment]
> Looks like method to get list of tasks is not implemented?

@mirkokg that is correct. The only methods that are implemented are those for which  we have use cases, and at the moment we don't have any use case for retrieving a list of tasks.
